src/Octoshift/Services/AdoApi.cs

```diff
@@ -602,7 +624,20 @@ public virtual async Task<string> GetBoardsGithubRepoId(string org, string teamP
         var response = await _client.PostAsync(url, payload);
         var data = JObject.Parse(response);
 
-        return (string)data["dataProviders"]["ms.vss-work-web.github-user-repository-data-provider"]["additionalProperties"]["nodeId"];
+        // Check for error message in the response
+        var dataProvider = data["dataProviders"]["ms.vss-work-web.github-user-repository-data-provider"];
+        var errorMessage = (string)dataProvider?["errorMessage"];
+        if (!string.IsNullOrEmpty(errorMessage))
+        {
+            throw new OctoshiftCliException($"Error getting GitHub repository information: {errorMessage}");
+        }
+
+        if (dataProvider == null || dataProvider["additionalProperties"] == null || dataProvider["additionalProperties"]["nodeId"] == null)
```

Comment:
The linter is throwing a warning (IDE0028) saying that this if statement can be simplified, but I like it as it is. Can you disable the linter warning for this one specific if statement.

Reply:
My mistake, that should have been IDE0046 not IDE0028